lib/scripts/parent-require.js

```diff
@@ -0,0 +1,9 @@
+module.exports = function (id) {
```

Comment:
move this to `lib` and credit the original author https://github.com/jaredhanson/node-parent-require

Reply:
to be specific: credit him in a code comment at the top of the file